lib/matplotlib/pyplot.py

```diff
@@ -3447,7 +3457,7 @@ def triplot(*args, **kwargs):
 @_autogen_docstring(Axes.violinplot)
 def violinplot(dataset, positions=None, vert=True, widths=0.5, showmeans=False,
```

Comment:
This one is slightly awkward, as I seem to remember "dataset" is a poor-man's multivariate datamodel (aka. Series). We should note this, and try to improve the interface in the future.